python/lsst/pipe/drivers/multiBandDriver.py

```diff
@@ -299,6 +310,20 @@ def run(self, patchRefList):
                     reprocessDataset + "_filename", dataId)[0]
                 os.unlink(filename)
 
+    def runDetection(self, cache, patchRef):
+        """! Run detection on a patch
+        Only slve nodes execute this method.
+
+        @param cache: Pool cache, containing butler
+        @param patchRef: Patch on which to do detection
+        """
+        with self.logOperation("do detections on {}".format(patchRef.dataId)):
+            idFactory = self.detectCoaddSources.makeIdFactory(patchRef)
+            self.log.info("%s: Reading coadd %s" % (NODE, patchRef.dataId))
+            coadd = patchRef.get(cache.coaddType, immediate=True)
```

Comment:
That is going to change again in the future I believe. It was only set that way for the pybind port temporarily.